src/System.Threading.Thread/src/Configurations.props

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      uap101aot-Windows_NT;
+      netstandard1.7-Windows_NT;
+      netstandard1.7-Unix;
+      net463-AnyOS;
```

Comment:
I know you just copied this from the builds file as is but all the net4x configurations should be Windows_NT to avoid building them cross-plat.